openmdao/core/tests/test_scaling_report_mpi.py

```diff
@@ -0,0 +1,14 @@
+"""Define the units/scaling tests."""
+import unittest
+
+from openmdao.utils.testing_utils import use_tempdirs
+from openmdao.core.tests.test_scaling_report import TestDriverScalingReport
+
+
+class TestDriverScalingReportMPI(TestDriverScalingReport):
+    N_PROCS = 2
+    ISOLATED = True
```

Comment:
I forgot somehow when I suggested using ISOLATED that the test was already an MPI test.  TestCases with N_PROCS defined already run isolated in their own subprocess, so the ISOLATED thing shouldn't be necessary.  Just moving the test to its own file is probably what got rid of the hang.